src/pages/pupils/l/[redirectFrom]/lessons/[lessonSlug]/index.tsx

```diff
@@ -83,10 +85,21 @@ export const getStaticProps: GetStaticProps<
           lessonSlug,
         })
         .catch((error) => {
-          console.error("Error fetching pupilLessonOverviewCanonical", {
-            lessonSlug,
-            error,
+          const forwardError: OakError = { ...error };
+          if (forwardError.code === "curriculum-api/not-found") {
+            forwardError.config.shouldNotify = true;
+          }
+
+          errorReporter(
+            "pupils::lesson-overview-legacy-canonical::getStaticProps::pupilLessonOverviewCanonical",
+          )(forwardError, {
+            severity: "warning",
+            ...{
+              lessonSlug,
+              redirectFrom,
+            },
```

Comment:
:surfer: the spread is redundant

```suggestion
            severity: "warning",
            lessonSlug,
            redirectFrom,
```

Reply:
you know I copied this from your code ;)